doc/linked-cs.md

```diff
@@ -0,0 +1,97 @@
+- *webResourcePrefix*  - definuje prefix webových zdrojů.
+
+Následuje příklad konfigurace serveru:
+
+```turtle
+@prefix conf: <http://richard.cyganiak.de/2007/pubby/config.rdf#> .
+
+<>
+  a conf:Configuration ;
+  conf:projectName "KBSS Ontologies" ;
+  conf:projectHomepage <http://onto.fel.cvut.cz/ontologies> ;
+  conf:webBase <http://onto.fel.cvut.cz/ontologies/> ;
+  conf:usePrefixesFrom <> ;
+  conf:defaultLanguage "en" ;
+  conf:webResourcePrefix "" .
+```
+V tomto příkladu je názvem projektu "KBSS Ontologies" a jeho domovskou stránkou je adresa http://onto.fel.cvut.cz/ontologies. URL adresy jednotlivých zdrojů jsou sestavovány ze základního URL http://onto.fel.cvut.cz/ontologies/, například [https://onto.fel.cvut.cz/ontologies/page/slovník/datový/mpp-3.5-np/pojem/typ-struktury](https://onto.fel.cvut.cz/ontologies/page/slovn%C3%ADk/datov%C3%BD/mpp-3.5-np/pojem/typ-struktury). |Jsou použity prefixy z tohoto konfiguračního dokumentu výchozím jazykem je angličtina.
+
+### Konfigurace datových sad
+
+Datové sady jsou v turtle vlastností konfigurace serveru `conf:dataset` a obsahují následující vlastnosti:
+
+- *sparqlEndpoint*  - URL přístupového bodu SPARQL, na kterém se nachází data datové sady, jedná se tedy o přístupový bod triple storu, do kterého TermIt ukládá data,
+- *datasetBase* - základní prefix datové sady (ten, který je použit jako @prefix v RDF datech datové sady).
+
+Příkladem nastavení datové sady je:
+
+```turtle
+<>
+  a conf:Configuration ;
+  ...
+	conf:dataset
+	  [
+		 conf:sparqlEndpoint <https://onto.fel.cvut.cz:7200/repositories/termit-dev> ;
+	     conf:datasetBase <http://onto.fel.cvut.cz/ontologies/>
+	  ].
+```
+Datová sada definovaná v tomto příkladu se nachází na SPARQL přístupovém bodu na URL adrese https://onto.fel.cvut.cz:7200/repositories/termit-dev a jako základní URL pro zdroje z této datové sady je použito http://onto.fel.cvut.cz/ontologies/.
+
+
+
+### Příklad konfiiguračního souboru
+```turtle
+@prefix conf: <http://richard.cyganiak.de/2007/pubby/config.rdf#> .
+
+<>
+  a <http://richard.cyganiak.de/2007/pubby/config.rdf#Configuration> ;
+  conf:projectName "KBSS Ontologies" ;
+  conf:projectHomepage <http://onto.fel.cvut.cz/ontologies> ;
+  conf:webBase <http://onto.fel.cvut.cz/ontologies/> ;
+  conf:usePrefixesFrom <> ;
+  conf:defaultLanguage "en" ;
+  conf:webResourcePrefix "" ;
+
+  conf:dataset
+ # 14GISON - IPR Datasets
+  [
+    conf:sparqlEndpoint <https://onto.fel.cvut.cz:7200/repositories/ipr_datasets> ;
+    conf:datasetBase <http://onto.fel.cvut.cz/ontologies/>
+  ],
+  [
+    conf:sparqlEndpoint <http://onto.fel.cvut.cz/rdf4j-server/repositories/cz-vugtk> ;
+    conf:datasetBase <http://onto.fel.cvut.cz/ontologies/>
+  ],
+
+  conf:metadataTemplate "metadata.ttl" .
+```
+Tento konfigurační soubor sdružuje do jedné datové sady zdroje nacházející se na dvou SPARQL přístupových bodech. Vlastnost `conf:metadataTemplate` odkazuje na umístění soubory s šablonou pro metadata zdrojů.
```

Comment:
na umístění soubory => to je nějaká divná věta

Reply:
that is weird, most of those typos were repaired yesterday before commit. let me take a look